src/webapp/components/button/Button.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Button as MUIButton } from "@material-ui/core";
+import styled from "styled-components";
+
+type ButtonProps = {
+    children?: React.ReactNode;
+    variant?: "contained" | "outlined";
+    color?: "primary" | "secondary" | "dark-secondary";
```

Comment:
It's because for example buttons from the page Incident Management Team Builder are different from the edit buttons in Incident Action Plan  or Event TRacker page. Maybe the design was made by someone else and that's why they are different in that pages, but maybe for more coherence it would be better to leave it in only two styles?

Reply:
makes sense

Reply:
removed!